### PR TITLE
Move bi-lib core to front end for nix

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
Previously a published npm package was used, i have moved bi-lib core as a dependancy package from turbo repo.

### CODE REVIEW

File: frontend/apps/picasso/package.json [hunk 1]
```diff
@@ -47,6 +46,7 @@
     "react-hotjar": "^4.0.0",
     "substrate-react": "*",
     "zustand": "^4.0.0-beta.2",
+    "@integrations-lib/core": "*",
```
Comment (easteregg) [inline frontend/apps/picasso/package.json:49]: Can we rename this to `bi-lib` to be consistent with the folder names?

File: frontend/packages/bi-lib/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (easteregg) [inline frontend/packages/bi-lib/tsconfig.json:1]: Can we extend from the tsconfig package and modify options here?

**Review by hussein-aitlahcen — APPROVED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Review by PoisonPhang — APPROVED** [review]

**Review by easteregg — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**